cli/commands/project.go

```diff
@@ -42,6 +42,16 @@ func (cmd *Project) Commands() []cli.Command {
 	return []cli.Command{command}
 }
 
+// Run before all commands to setup core services
+func (cmd *Project) Before(c *cli.Context) error {
+	if err := cmd.BaseCommand.Before(c); err != nil {
+		return err
+	}
+	cmd.out.Verbose.Printf("Loaded project configuration from %s", cmd.Config.Path)
```

Comment:
Why not just print this out at the start of Run()?

Reply:
The idea is that for a given file we want every command to load and log configuration the same way. This seemed slightly DRY-er. I briefly explored the idea of having the project set a flag to enable config and do this just in BaseCommand but pulled back on the basis of scope.

Reply:
For the project command, the config is loaded in Commands() so the config is loaded already and every script passes through Run() before it gets executed, so I would think either moving the log message to Run() so we dont need the BaseCommand delegation or move the log to Commands() after it is loaded to the load/log are in the same place (I realize in Commands() you dont have access to the verbose flag, so think moving to Run() is the right call)